packages/web-app-ocm/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import App from './views/App.vue'
+import { defineWebApplication, useRouter } from '@ownclouders/web-pkg'
+import translations from '../l10n/translations.json'
+import { extensions } from './extensions'
+import { RouteRecordRaw } from 'vue-router'
+import { useGettext } from 'vue3-gettext'
+
+const appInfo = {
+  name: 'ScienceMesh',
```

Comment:
please wrap the name in a `$gettext` dummy or move it into the `defineWebApplication` body.